src/c++11/recorder.cc

```diff
@@ -110,7 +110,9 @@ BasicRecorder::BasicRecorder(const TracerImpl& tracer, const BasicRecorderOption
     flushed_seqno_(0),
     encoding_seqno_(1),
     dropped_spans_(0),
-    client_(grpc::CreateChannel(urlOf(tracer.options()),
+    client_(grpc::CreateChannel(hostPortOf(tracer.options()),
+				(tracer.options().collector_encryption == "tls") ?
+				grpc::SslCredentials(grpc::SslCredentialsOptions()) :
```

Comment:
It parallels gRPC initialization in the other languages, but I haven't tested reporting to an SSL endpoint yet.